tests/e2e/support/objects/app-files/share/index.ts

```diff
@@ -68,6 +71,11 @@ export class Share {
 
   async copyQuickLink(args: Omit<copyLinkArgs, 'page'>): Promise<void> {
     await copyQuickLink({ ...args, page: this.#page })
+    const quickLink = await this.#page.evaluate(() => navigator.clipboard.readText())
+    this.#linksEnvironment.createLink({
+      key: 'Quicklink',
+      link: { name: 'Quicklink', url: quickLink }
+    })
```

Comment:
i think this action is actually createAndCopyQuickLink

Reply:
The `copyQuickLink` method is used to copy links through UI, which won't create links every time. If there is no quick link created then it will create a new link and copy the link otherwise it will just copy the link. This might fail when the same step is used more than 1 time.

Reply:
@grgprarup  Isn't `copyQuickLink` just used to copy the quick links? In that case, it should work because you cannot create two quick links at once. Whenever you click the quick link button the quick link either already created or newly created is copied. I don't understand what you're trying to say 😅

Reply:
```
this.#linksEnvironment.createLink({
      key: 'Quicklink',
      link: { name: 'Quicklink', url: quickLink }
    })
```

This will try to create a link every time and step `And "Brian" copies quick link of the resource "shared_folder" from the context menu` might fail if a step is used more than 1 time in a single scenario.

Reply:
yes true

Reply:
Before adding the link to the link store we might want to check if the link store has the link with the key `quick-link` and if the key exists don't add it to the link store.

Reply:
we should also clear the link store after the execution of the scenario